atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraTimestampBoundStore.java

```diff
@@ -155,6 +163,12 @@ private void cas(Client client, Long oldVal, long newVal) {
         }
     }
 
+    private String getCurrentValues(CASResult result) {
+        List<Long> timestamps = result.current_values.stream().map(col -> PtBytes.toLong(col.getValue())).collect(
+                Collectors.toList());
+        return StringUtils.join(timestamps, ',');
```

Comment:
Should probably use the `Collectors.joining(", ")` instead (with a space after the comma.